src/views/poi_panel.dot

```diff
@@ -119,7 +119,7 @@
             {{~ this.categories:category:index }}
               <button class="service_panel__category" type="button" {{= click(this.openCategory, this, category) }}>
                 <div class="service_panel__category__icon" style="background: {{= category.backgroundColor }}">
-                  <span class="icon icon-{{= category.iconName }}"></span>
+                  <span class="{{? index < 9 || index == 10 }}icon{{??}}category__new-icon{{?}} icon-{{= category.iconName }}"></span>
```

Comment:
Not sure I understand this rule… Why is it using a new CSS class?

Reply:
Yeah, it's a bit dirty for now, as we discussed on the map Thread last week.
9 of the icons are present in the icon font shared with the map (class=icon)
3 new icons are only present in the icomoon font (class=new_icon)
It's agreed to merge like that for now, and add the 3 new icons in the shared font when we have the time (Adrien needs to teach us how to do it)

Reply:
Alright! Thanks for the explanation :)